client/src/CommentsContainer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Map, Set } from 'immutable';
+
+export default function CommentsContainer(props) {
+  const {bit, comments = Map()} = props;
+  return (
+    <div>
+      {bit.get('comments', Set()).map(commentId => {
+        const comment = comments.get(commentId, Map());
+        return (
+          <div key={commentId}>
+            <p>
+              <b>{comment.getIn(['author', 'name'])}</b>
+              {` \u2022 `}
```

Comment:
what... is this line?

Also aren't we using utf-8 already? We should be able to just put the unicode character here instead of a code.

Reply:
Done.